spec/unit/controllers/v3/service_instances_controller_spec.rb

```diff
@@ -23,6 +23,28 @@
           response_names = parsed_body['resources'].map { |resource| resource['name'] }
           expect(response_names).to include(service_instance.name, service_instance2.name, service_instance3.name)
         end
+
+        context 'when service instances are filtered by name' do
+          it 'returns only the matching service instances' do
+            get :index, names: 'service-instance-1,service-instance-2'
+            expect(response.status).to eq(200), response.body
+            expect(parsed_body['resources'].length).to eq 2
+
+            response_names = parsed_body['resources'].map { |resource| resource['name'] }
+            expect(response_names).to include(service_instance.name, service_instance2.name)
+          end
+        end
+
+        context 'when service instances are filtered by space guid' do
+          it 'returns only the matching service instances' do
+            get :index, space_guids: 'space-2-guid, space-3-guid'
```

Comment:
There is an extra space between `space-2-guid` and `space-3-guid`

Reply:
Or, is that intentional? Do we test cases like this elsewhere?

Reply:
We wanted to test that the parsing logic ignores whitespaces at some point, but this probably isn't the right place for that. We'll change it to make it consistent.

Reply:
Changes for this included in #1033 and can be viewed here: https://github.com/cloudfoundry-incubator/cloud_controller_ng_sapi/commit/ff8276e5d201e573df03640a0f6865288b2d55af#diff-eb9dfb65b6b62e5646cab0d740db45e3R41